intergration/radon-test/r/desc_explain.result

```diff
@@ -41,4 +41,17 @@ c1	int(11)	NO	PRI	NULL
 c2	int(11)	YES		NULL	
 
 
+explain select * from integrate_test.t;
```

Comment:
remove `\n` and `\t`.
such as:
```
{\n\t"RawQuery": "explain select * from integrate_test.t",
```
instead of:
```
{
    "RawQuery": "explain select * from integrate_test.t",
```

Reply:
Done.